sympy/combinatorics/coset_table.py

```diff
@@ -527,31 +544,25 @@ def coincidence(self, alpha, beta, w=None,modified=False):
                 delta = table[gamma][A_dict[x]]
                 if delta is not None:
                     table[delta][A_dict_inv[x]] = None
-                    mu = None
-                    nu = None
-                    if modified:
-                        mu = self.modified_rep(gamma, p_p)
-                        nu = self.modified_rep(delta, p_p)
-                    else:
-                        mu = self.rep(gamma)
-                        nu = self.rep(delta)
+                    mu = self.rep(gamma, modified=modified)
+                    nu = self.rep(delta, modified=modified)
                     if table[mu][A_dict[x]] is not None:
                         if modified:
-                            v = p_p[delta]**-1*self.P[gamma][self.A_dict[x]]**-1*p_p[gamma]*self.P[mu][self.A_dict[x]]
-                            self.modified_merge(nu, table[mu][self.A_dict[x]], v, p_p, q)
+                            v = self.p_p[delta]**-1*self.P[gamma][self.A_dict[x]]**-1*self.p_p[gamma]*self.P[mu][self.A_dict[x]]
```

Comment:
This line is too long - it should be shortened to <= 72 symbols (generally, if you run `git diff` in terminal before committing the changes, the lines that are too long will wrap around - that's how you tell).